client/features/checkout/CheckoutReview.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import Button from "@mui/material/Button";
+import { usdFormatter, getShippingCost } from "../../helpers";
+import { Typography, Container } from "@mui/material";
+import { makeStyles } from "@material-ui/core";
+
+const dollar = (x) => {
```

Comment:
Looks like this was copied from an old version of `CartSummary` maybe. I moved the function `dollar` into `helpers.js`. Lets refactor to use that function throughout